pkg/controller/certificatemanager/certificatemanager.go

```diff
@@ -110,17 +114,17 @@ func Create(cli client.Client, installation *operatorv1.InstallationSpec, cluste
 		}
 		certificatePEM = certificateManagement.CACert
 	} else {
+		// Using operator-managed certificates. Check to see if we have already provisioned one.
 		caSecret := &corev1.Secret{}
-		err := cli.Get(context.Background(), types.NamespacedName{
-			Name:      certificatemanagement.CASecretName,
-			Namespace: common.OperatorNamespace(),
-		}, caSecret)
-		if err != nil && !kerrors.IsNotFound(err) {
+		k := types.NamespacedName{Name: certificatemanagement.CASecretName, Namespace: ns}
+		if err = cli.Get(context.Background(), k, caSecret); err != nil && !kerrors.IsNotFound(err) {
```

Comment:
We want a different CA for each tenant's namespace? What would be the advantage of that?

Reply:
The advantage being that each tenant's components will be issued certificates from their own CA - invalidating a single tenant becomes trivial (just delete the CA and regenerate it)

Reply:
I'm not convinced I see a use-case for that. Aren't we over-complicating things? The tenant's components are still just our components inside of our cluster, they just happen to serve a particular tenant. 

I'd argue if we want isolation of these tenant namespaces, we should be using network policies.

Reply:
> I'd argue if we want isolation of these tenant namespaces, we should be using network policies.

We'll do both. I agree that it's not a requirement - a single CA for all services would work as well, but we need a separate CA for each tenant's Voltron mTLS tunnel anyway, so using that CA for other services seems like an easy defense-in-depth improvement.